conda_libmamba_solver/repoquery.py

```diff
@@ -46,6 +55,11 @@ def configure_parser(parser: argparse.ArgumentParser):
         action="store_true",
         help="Look at all channels (for depends / whoneeds).",
     )
+    package_grp.add_argument(
+        "--use-cache-only",
+        action="store_true",
+        help="Search in pkgs_dirs too",
+    )
```

Comment:
They have changed quite a few arguments for v2. Some of them don't seem to be part of the `repoquery` CLI actually:

```
$ mamba repoquery depends --help
List dependencies of a given query
Usage: mamba repoquery depends [OPTIONS] specs...

Positionals:
  specs TEXT ... REQUIRED     Specs to search

Options:
  -h,--help                   Print this help message and exit
  -c,--channel TEXT ...       Define the list of channels
  --override-channels         Override channels
  --channel-priority ENUM:value in {disabled->0,flexible->1,strict->2} OR {0,1,2}
                              Define the channel priority ('strict' or 'disabled')
  --channel-alias TEXT        The prepended url location to associate with channel names
  --strict-channel-priority   Enable strict channel priority
  --no-channel-priority       Disable channel priority
  -t,--tree                   Show result as a tree
  --recursive                 Show dependencies recursively, i.e. transitive dependencies (only for `depends`)
  --pretty                    Pretty print result (only for search)
  --platform TEXT             The platform description
  --remote                    Use installed prefix instead of remote repositories.


Configuration options:
  --rc-file TEXT ...          Paths to the configuration files to use
  --no-rc                     Disable the use of configuration files
  --no-env                    Disable the use of environment variables


Global options:
  -v,--verbose                Set verbosity (higher verbosity with multiple -v, e.g. -vvv)
  --log-level ENUM:value in {critical->5,debug->1,error->4,info->2,off->6,trace->0,warning->3} OR {5,1,4,2,6,0,3}
                              Set the log level
  -q,--quiet                  Set quiet mode (print less output)
  -y,--yes                    Automatically answer yes on prompted questions
  --json                      Report all output as json
  --offline                   Force use cached repodata
  --dry-run                   Only display what would have been done
  --download-only             Only download and extract packages, do not link them into environment.
  --experimental              Enable experimental features


Prefix options:
  -r,--root-prefix TEXT       Path to the root prefix
  -p,--prefix TEXT            Path to the target prefix
  --relocate-prefix TEXT      Path to the relocation prefix
  -n,--name TEXT              Name of the target prefix


Network options:
  --ssl-verify TEXT           Verify SSL certificates for HTTPS requests
  --ssl-no-revoke             SSL certificate revocation checks
  --cacert-path TEXT          Path (file or directory) SSL certificate(s)
  --repodata-ttl UINT         Repodata time-to-live
  --retry-clean-cache         If solve fails, try to fetch updated repodata
```

Reply:
This is for 1.x:

```
$ micromamba repoquery search --help
Find and analyze packages in active environment or channels
Usage: micromamba repoquery [OPTIONS] query_type specs...

Positionals:
  query_type TEXT:{search,depends,whoneeds} REQUIRED
                              The type of query (search, depends or whoneeds)
  specs TEXT ... REQUIRED     Specs to search

Options:
  -h,--help                   Print this help message and exit
  -c,--channel TEXT ...       Define the list of channels
  --override-channels         Override channels
  --channel-priority ENUM:value in {disabled->0,flexible->1,strict->2} OR {0,1,2}
                              Define the channel priority ('strict' or 'disabled')
  --channel-alias TEXT        The prepended url location to associate with channel names
  --strict-channel-priority   Enable strict channel priority
  --no-channel-priority       Disable channel priority
  -t,--tree                   Show result as a tree
  --recursive                 Show dependencies recursively, i.e. transitive dependencies (only for `depends`)
  --pretty                    Pretty print result (only for search)
  --use-local INT             Use installed data (--use-local=1, default for `depends` and `whoneeds`) or remote repositories (--use-local=0, default for `search`).
                              If the `-c,--channel` option is set, it has the priority and --use-local is set to 0
  --platform TEXT             The platform description


Configuration options:
  --rc-file TEXT ...          Paths to the configuration files to use
  --no-rc                     Disable the use of configuration files
  --no-env                    Disable the use of environment variables


Global options:
  -v,--verbose                Set verbosity (higher verbosity with multiple -v, e.g. -vvv)
  --log-level ENUM:value in {critical->5,debug->1,error->4,info->2,off->6,trace->0,warning->3} OR {5,1,4,2,6,0,3}
                              Set the log level
  -q,--quiet                  Set quiet mode (print less output)
  -y,--yes                    Automatically answer yes on prompted questions
  --json                      Report all output as json
  --offline                   Force use cached repodata
  --dry-run                   Only display what would have been done
  --download-only             Only download and extract packages, do not link them into environment.
  --experimental              Enable experimental features


Prefix options:
  -r,--root-prefix TEXT       Path to the root prefix
  -p,--prefix TEXT            Path to the target prefix
  --relocate-prefix TEXT      Path to the relocation prefix
  -n,--name TEXT              Name of the target prefix


Network options:
  --ssl-verify TEXT           Verify SSL certificates for HTTPS requests
  --ssl-no-revoke             SSL certificate revocation checks
  --cacert-path TEXT          Path (file or directory) SSL certificate(s)
  --repodata-ttl UINT         Repodata time-to-live
  --retry-clean-cache         If solve fails, try to fetch updated repodata
```